HW_1/task_3.py

```diff
@@ -0,0 +1,20 @@
+# 3. Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+while True:
+    num = int(input('Enter number 2-100000: '))
+    if 2 > num or num > 100000:
+        print('Try again!!!: ')
+    else:
+        print(num)
+        if 1 < num < 4:
+            print('Prime number!')
+        else:
+            for i in range(2, (num//2)+1):
```

Comment:
```suggestion
            for i in range(2, num // 2+1):
```
Но все равно слишком много итераций. Вспомните теорему из теории чисел, которая гласит, что для любого составного числа есть, по крайней мере, один делитель, не превосходящий корень из этого числа.